### PR TITLE
Bump version and CHANGELOG for v3.87.0

### PR DESCRIPTION
**Pull request by CerealBoy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.87.0](https://github.com/buildkite/agent/tree/v3.87.0) (2024-11-18)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.86.0...v3.87.0)
+
+### Added
+- Switch to agent-base images [#3091](https://github.com/buildkite/agent/pull/3091) (@DrJosh9000)
```
Comment (DrJosh9000) [inline CHANGELOG.md:12]: Internal? I doubt anyone cares but us.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
...
+
+### Changed
+- Remove signal reason unable\_to\_verify\_signature and replace with signature\_rejected [#3094](https://github.com/buildkite/agent/pull/3094) (@jordandcarter)
+- Soft fail upload of packages docker images [#3093](https://github.com/buildkite/agent/pull/3093) (@tommeier)
```
Comment (DrJosh9000) [inline CHANGELOG.md:16]: Also Internal?

**Review by DrJosh9000 — APPROVED** [review]
🗒️

File: CHANGELOG.md (hunk 1 above)
Comment (CerealBoy) [reply CHANGELOG.md:12]: Oh yeh, good point

**Pull request merged**